semgrep_output_v1.py

```diff
@@ -3379,6 +3379,91 @@ def to_json_string(self, **kw: Any) -> str:
         return json.dumps(self.to_json(), **kw)
 
 
+@dataclass(frozen=True)
+class Error_:
```

Comment:
Not super relevan to this PR, but this name collision seems annoying to deal with. Is there a reason ATD generates the python like this instead of using [`enum.Enum`](https://docs.python.org/3/library/enum.html)?

Reply:
cc @mjambon on why not using Enum. My guess is that using class generalizes better to support ADTs, so if Error suddently takes some parameter (e.g., a position).